packages/eas-cli/src/credentials/ios/appstore/provisioningProfile.ts

```diff
@@ -7,8 +8,68 @@ import {
   ProvisioningProfileStoreInfo,
 } from './Credentials.types';
 import { AuthCtx } from './authenticate';
+import { transformCertificate } from './distributionCertificate';
+import { USE_APPLE_UTILS, getBundleIdForIdentifier, getProfilesForBundleId } from './experimental';
 import { runActionAsync, travelingFastlane } from './fastlane';
 
+async function transformProfileAsync(
+  cert: Profile,
+  ctx: AuthCtx
+): Promise<ProvisioningProfileStoreInfo> {
+  return {
+    provisioningProfileId: cert.id,
+    name: cert.attributes.name,
+    status: cert.attributes.profileState,
+    expires: new Date(cert.attributes.expirationDate).getTime() / 1000,
+    distributionMethod: cert.attributes.profileType,
+    // @ts-ignore -- this can be null when the profile has expired.
+    provisioningProfile: cert.attributes.profileContent,
+    certificates: (await cert.getCertificatesAsync()).map(transformCertificate),
+    teamId: ctx.team.id,
+    teamName: ctx.team.name,
+  };
+}
+
+async function getCertificateBySerialNumberAsync(
+  distCertSerialNumber: string
+): Promise<Certificate> {
+  const cert = (
+    await Certificate.getAsync({
+      query: { filter: { serialNumber: [distCertSerialNumber] } },
+    })
+  ).find(item => item.attributes.serialNumber === distCertSerialNumber);
+  if (!cert) {
+    throw new Error(
+      'No cert available to make provision profile against. Make sure you were able to make a certificate prior to this step'
+    );
+  }
+  return cert;
+}
```

Comment:
I think this should be moved to `distributionCertificate.ts`. 
Remember to update the error message afterward.